webapp/backend/src/handlers/api/tests/videos.spec.ts

```diff
@@ -29,8 +29,8 @@ describe("Demo video routes", () => {
 
   it("should get response from demo videos metadata with id", async () => {
     try {
-        const response = await request.get(`/${id}/metadata`);
-        expect(response.status).toBe(200);
+        const response = await request.get(`/${id})/metadata`);
+        expect(response.status).toBe(400);
```

Comment:
this looks like a negative test? if so please modify the test description on line 30

Reply:
please add tests for empty, `NaN` and negative id as well since they are checks in the code in `videos.ts`